### PR TITLE
changes to upgrade to ocata

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: docker/postlaunchconfig_update.sh [hunk 1]
```diff
@@ -170,5 +171,10 @@ openstack-config --set /etc/keystone/keystone.conf \
...
 
 # Ensure db is migrated to current keystone version
-/usr/bin/keystone-manage db_sync
-/usr/bin/keystone-manage db_sync --extension spassword
+echo "[ postlaunchconfig_update - db_sync --migrate ] "
+/usr/bin/keystone-manage db_sync --migrate
+
+
+kill -9 $keystone_all_pid
+#kill -9 $keystone_admin_pid
```
Comment (fgalan) [inline docker/postlaunchconfig_update.sh]: Lefover?

File: docker/Dockerfile [hunk 2]
```diff
@@ -68,7 +71,7 @@ RUN \
     cd ${dir}/${repo_scim}-${tag} && \
     source ./${pack_scim} --with-python27 --with-version $KEYSTONE_SCIM_VERSION --with-release 0 && \
     find . -name "*.rpm" -exec rpm -Uvh {} \; && \
-    # Install Keystone-SPASSWORD
+     # Install Keystone-SPASSWORD
```
Comment (fgalan) [inline docker/Dockerfile]: Miss indented?

File: docker/postlaunchconfig_update.sh (hunk 1 above)
Comment (AlvaroVega) [reply docker/postlaunchconfig_update.sh]: fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/136/commits/25a077d47c9688b78c434359e78411fb11a6ba39

File: docker/Dockerfile (hunk 2 above)
Comment (AlvaroVega) [reply docker/Dockerfile]: fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/136/commits/25a077d47c9688b78c434359e78411fb11a6ba39

**Pull request merged**